src/js/view/controlbar.js

```diff
@@ -464,6 +470,19 @@ define([
                     this.elements.right.insertBefore(ele.el, this.elements.drawer.el);
                 }, this);
             }
+        },
+        rewind : function() {
+            var currentPosition = this._model.get('position'),
+                duration = this._model.get('duration'),
+                rewindPosition = currentPosition - 10,
+                startPosition = 0;
+
+            // duration is negative in DVR mode
+            if (utils.adaptiveType(duration) === 'DVR') {
+                startPosition = duration;
+            }
+            // Seek 10s back. Seek value should be >= 0 in VOD mode and >= (negative) duration in DVR mode
+            this._api.seek(Math.max(rewindPosition, startPosition));
```

Comment:
It would be nice to have unit tests for this, if its quick and easy to setup a test file for controlbar.js